src/parquet/arrow/writer.cc

```diff
@@ -717,6 +717,10 @@ Status FileWriter::WriteTable(const Table& table, int64_t chunk_size) {
     }
   }
 
+  if (chunk_size <= 0) {
+    return Status::Invalid("chunk size per row_group must be greater than 0");
```

Comment:
Can you test that error status is returned when chunk_size is equal to 0? Better than segfault or infinite loop